environments/stg/stg.tfvars

```diff
@@ -1282,5 +1282,13 @@ frontends = [
     backend_domain   = ["firewall-prod-int-palo-hmiapimaat.uksouth.cloudapp.azure.com"]
     certificate_name = "wildcard-staging-platform-hmcts-net"
     cache_enabled    = "false"
+  },
+  {
+    product          = "cft-api-mgmt"
+    name             = "cft-api-mgmt"
+    custom_domain    = "cft-api-mgmt.aat.platform.hmcts.net"
+    backend_domain   = ["firewall-prod-int-palo-cftapimgmtstg.uksouth.cloudapp.azure.com"]
+    certificate_name = "wildcard-aat-platform-hmcts-net"
```

Comment:
hmm prefer managed certs, if you can

Reply:
Updated to its own managed cert - cft-api-mgmt-aat-platform-hmcts-net